integration/tests/cook/util.py

```diff
@@ -228,7 +229,7 @@ def wait_until_inner():
         return wait_until_inner()
     except:
         final_response = query()
-        logger.info(f"Timeout exceeded waiting for condition. Details: {final_response}")
+        logger.info(f"Timeout exceeded waiting for condition. Details: {final_response.content}")
```

Comment:
Sometimes `final_response` isn't actually a response. Can you add a guard here and only log `.content` if it actually exists?

Reply:
Done.